server/static/js/student.js

```diff
@@ -0,0 +1,19 @@
+jQuery(document).ready(function($){
+    $('.remove-confirm').on('click',function(e){
+        e.preventDefault();
+        var form = $(this).parents('form');
+        var info = form[0][1].value;
+        swal({
+            title: "Just double checking",
+            text: "Are you sure you want to remove " + info + " from the group?",
```

Comment:
If you try to leave, it shows "Are you sure you want to remove [your email] from the group?", which could be confusing.